src/components/Updates/FinancialSection.tsx

```diff
@@ -0,0 +1,188 @@
+import { useCallback, useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { usePapaParse } from 'react-papaparse'
+
+import toNumber from 'lodash/toNumber'
+
+import { UpdateFinancial, UpdateFinancialRecord, UpdateFinancialSchema } from '../../entities/Updates/types'
+import useFormatMessage from '../../hooks/useFormatMessage'
+import CSVDragAndDrop from '../Common/CSVDragAndDrop'
+import NumberedTextArea from '../Common/NumberedTextArea'
+import SubLabel from '../Common/SubLabel'
+import Label from '../Common/Typography/Label'
+import { ContentSection } from '../Layout/ContentLayout'
+import ProjectRequestSection from '../ProjectRequest/ProjectRequestSection'
+
+import './FinancialSection.css'
+import SummaryItems from './SummaryItems'
+
+interface Props {
+  onValidation: (data: UpdateFinancial, sectionValid: boolean) => void
+  isFormDisabled: boolean
+  sectionNumber: number
+  intialValues?: UpdateFinancial
+}
+
+const UPDATE_FINANCIAL_INITIAL_STATE: UpdateFinancial = {
+  records: [],
+}
+
+const CSV_HEADER: (keyof UpdateFinancialRecord)[] = [
+  'concept',
+  'description',
+  'amount',
+  'token_type',
+  'receiver',
+  'link',
+]
+
+const SEPARATOR = ','
+
+const schema = UpdateFinancialSchema
```

Comment:
I think we can drop this reassignment and use UpdateFinancialSchema directly, WDYT?